### PR TITLE
Feature/redesign about section

### PR DESCRIPTION
**Pull request by prachigarg19**
Current implementation:
image

Before:
image

### CODE REVIEW

**Comment by sunilsabatp** [comment]
Apply ` cursor:pointer` to the button

**Comment by sunilsabatp** [comment]
"I'm not sure, but it seems like some words have been merged into one.
image

**Comment by prachigarg19** [comment]
> Apply ` cursor:pointer` to the button

Are you talking about `See More` button, it already has this property

File: src/temp/AboutSection/AboutProject.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  amountOfWords: number;
+}
+
+const AboutProject = ({ description, amountOfWords }: Props) => {
+  const [isExpanded, setIsExpanded] = useState(false);
+  const splittedText = description.split(' ');
+
+  const itCanOverflow = splittedText.length > amountOfWords;
+  const beginText = itCanOverflow
+    ? splittedText.slice(0, amountOfWords - 1).join(' ')
```
Comment (sunilsabatp) [inline src/temp/AboutSection/AboutProject.tsx:17]: The logic for splitting the description into "beginText" and "endText" can be a bit convoluted. Consider simplifying it or adding comments to clarify its purpose.

**Review by sunilsabatp — APPROVED** [review]

**Comment by gangakhushlani** [comment]
Screenshot 2024-03-27 at 4 45 45 PM



CTA style is different here in about project box @sunilsabatp

**Review by gangakhushlani — COMMENTED** [review]
Change "see more" button style

File: src/temp/AboutSection/AboutProject.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    : description;
+  const endText = splittedText.slice(amountOfWords - 1).join(' ');
+  const { t } = useTranslation(['projectDetails', 'donate']);
+
+  return (
+    <div className={styles.projectDescription}>
+      <div className={styles.infoTitle}>{t('donate:aboutProject')}</div>
+      <div className={styles.infoText}>
+        {beginText}{' '}
```
Comment (mohitb35) [inline src/temp/AboutSection/AboutProject.tsx:26]: Add '...' when only the `beginText` is being shown (if there is `endText`)

File: src/temp/AboutSection/AboutProject.tsx
Comment (mohitb35) [inline src/temp/AboutSection/AboutProject.tsx:1]: Logic is fine, but I'd suggest some renaming for clarity.

1. `splittedText` can become `descriptionWords`
2. `amountOfWords` in props could become `wordCount`
3. `itCanOverflow` can become `hasLargeDescription` or `hasOverflow`
4. `beginText` can become `startingText` (begin is an action, so `startingText` indicates the purpose better)

**Review by mohitb35 — CHANGES_REQUESTED** [review]
@prachigarg19 I've left some minor feedback.

**Comment by prachigarg19** [comment]
> Change "see more" button style

Tis design change was suggested by @sagararyal

**Review by mohitb35 — APPROVED** [review]

**Pull request merged**